python/verticox/likelihood.py

```diff
@@ -148,37 +174,52 @@ def derivative_2_diagonal(z: ArrayLike, params: Parameters, u):
 
         first_part = np.square(params.K) * np.exp(params.K * z[u]) / denominator
 
-        second_part = np.square(params.K) * np.square(np.exp(params.K * z[u])) / np.square(
-            denominator)
+        second_part = np.square(params.K) * np.square(np.exp(params.K * z[u])) / \
+                      np.square(denominator)
 
-        summed += _get_dt(t, params.Dt) * (first_part - second_part)
+        summed += params.deaths_per_t[t] * (first_part - second_part)
 
     return summed + params.K * params.rho
 
 
-def derivative_2_off_diagonal(z: ArrayLike, params: Parameters, u, v):
+@numba.njit()
+def derivative_2_off_diagonal(z: ArrayLike, params, u, v):
     min_event_time = min(params.event_times[u], params.event_times[v])
     relevant_event_times = [t for t in params.Rt.keys() if t <= min_event_time]
```

Comment:
Does the numba jit handle this line properly? (python list comprehension)

Reply:
Yes: https://numba.pydata.org/numba-doc/dev/reference/pysupported.html#list-comprehension